catwalk/task.py

```diff
@@ -20,9 +19,6 @@
 
 ENTAILMENT_METRICS = {
     "acc": torchmetrics.Accuracy,
-    "f1": torchmetrics.F1Score,
-    "precision": torchmetrics.Precision,
-    "recall": torchmetrics.Recall
```

Comment:
If we make tasks aware of how many classes they have, it would be relatively easy to report F1 as well. That would be desirable right?

Reply:
done